compiler-core/src/language_server/tests/mod.rs

```diff
@@ -22,8 +20,10 @@ use crate::{
     Result,
 };
 
+mod compilation;
+
 #[derive(Debug, Clone, PartialEq, Eq)]
-enum Action {
+pub enum Action {
```

Comment:
Could you revert the changes to this file that make the definitions public? Thanks